newsroom/news_api/news/syndicate/auth.py

```diff
@@ -0,0 +1,18 @@
+from functools import wraps
+from flask import current_app as app
+import flask
+
+
+def authenticate(func):
```

Comment:
`authenticate` is a very generic name, would be good to have a name that indicates what type of authentication this wrapper provides